zkevm-circuits/src/rlp_circuit.rs

```diff
@@ -1222,11 +1222,6 @@ impl<F: Field> RlpCircuitConfig<F> {
                         meta.query_advice(rlp_table.data_type, Rotation::next()),
                         RlpDataType::TxHash.expr(),
                     );
-                    cb.require_equal(
-                        "TxHash rows' first row is Prefix again",
-                        meta.query_advice(rlp_table.tag, Rotation::next()),
-                        RlpTxTag::Prefix.expr(),
-                    );
```

Comment:
Please restore the deleted code and ensure it only runs when the "kroma" feature is deactivated.

Reply:
This is not equivalent to my change. For the non-deposit tx, the validation should be done.